src/Majora/Framework/Loader/LoaderTrait.php

```diff
@@ -12,148 +14,34 @@
  */
 trait LoaderTrait
 {
-    /**
-     * @var RepositoryInterface
```

Comment:
Pourquoi supprimer le docblock ? La classe `DoctrineLoaderTrait` qui utilise ce trait n'a pas de complétion sur la méthode `$this->entityRepository->retrieveAll` par exemple.

Ceci dit, elle ne l'a pas non plus en gardant ce docblock, car `retrieveAll` ne vient pas de `RepositoryInterface`. Il faudrait mieux pas documenter la bonne classe/interface ?